ecommerce/README.rst

```diff
@@ -0,0 +1,27 @@
+edx Themes for Otto
+===================
+This directory contains themes for Otto, following directory structure should be followed for each theme
+
+
+.. code-block:: text
+
```

Comment:
ditto about whether the "themes" directory should be added here.
Also, @mattdrayer if I understand your explanation of giving theme directories unique names, shouldn't an e-commerce theme directory have a different name than a Studio/LMS theme directory? or can they be the same?